kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiRestFrontendService.scala

```diff
@@ -240,7 +252,7 @@ class KyuubiRestFrontendService(override val serverable: Serverable)
       realUser
     } else {
       sessionConf.get(KyuubiAuthenticationFactory.HS2_PROXY_USER).map { proxyUser =>
-        if (!getConf.get(KyuubiConf.SERVER_ADMINISTRATORS).contains(realUser)) {
+        if (!isAdministrator(realUser)) {
```

Comment:
the behavior here has changed, previously, `Utils.currentUser` was not counted, which is inconsistent with checking administrator logic in `AdminResource`